pkgs/by-name/up/uplift/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "uplift";
+  version = "2.26.0";
+
+  src = fetchFromGitHub {
+    owner = "gembaadvantage";
+    repo = "uplift";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-cXZAxsQWjzYgoRAEKflJFcVN0OabdPHuJonJ0KMEaXs=";
+  };
+
+  vendorHash = "sha256-mjhNgy/QGHVdujmy94NQaoyvfrVmsIKZS3uWYK1XQr0=";
+
+  ldflags = with finalAttrs.src; [
+    "-s"
+    "-w"
+    "-X=github.com/gembaadvantage/uplift/internal/version.version=${tag}"
+    "-X=github.com/gembaadvantage/uplift/internal/version.gitCommit=${tag}"
```

```suggestion
  ldflags = [
    "-s"
    "-w"
    "-X=github.com/gembaadvantage/uplift/internal/version.version=v${finalAttrs.version}"
    "-X=github.com/gembaadvantage/uplift/internal/version.gitCommit=v${finalAttrs.version}"
```

Using `finalAttrs.src.tag` here would break when overriding the `src` with something that doesn't have a `tag` attribute (e.g. a local path).